sdk/core/azure-core/azure/core/pipeline/policies/_utils.py

```diff
@@ -67,3 +68,6 @@ def get_retry_after(response):
             parsed_retry_after = parse_retry_after(retry_after)
             return parsed_retry_after / 1000.0
     return None
+
+def get_domain(url):
+    return str(urlparse(url).netloc).lower()
```

Comment:
nit: I think netloc will always be a `str` even when empty

Reply:
The type of netloc is bytes. We need it to make our typing happy. :)

Reply:
Ah I remembered wrong , it's string if no value is present 

https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse